src/components/Calculator.js

```diff
@@ -33,45 +34,46 @@ class Calculator extends React.Component {
-        <div className="return-section">{(this.returnSection())}</div>
-        <div className="digits-elts">
-          <button type="button" className="button-section" value="AC" onClick={() => { this.calculator(digitObj, 'AC'); }}>AC</button>
-          <button type="button" className="button-section" value="+/-" onClick={() => { this.calculator(digitObj, '+/-'); }}>+/-</button>
-          <button type="button" className="button-section" value="%" onClick={() => { this.calculator(digitObj, '%'); }}>%</button>
-          <button type="button" className="button-section operator" value="÷" onClick={() => { this.calculator(digitObj, '÷'); }}>÷</button>
-          <button type="button" className="button-section" value="7" onClick={() => { this.calculator(digitObj, '7'); }}>7</button>
-          <button type="button" className="button-section" value="8" onClick={() => { this.calculator(digitObj, '8'); }}>8</button>
-          <button type="button" className="button-section" value="9" onClick={() => { this.calculator(digitObj, '9'); }}>9</button>
-          <button type="button" className="button-section operator" value="x" onClick={() => { this.calculator(digitObj, 'x'); }}>x</button>
-          <button type="button" className="button-section" value="4" onClick={() => { this.calculator(digitObj, '4'); }}>4</button>
-          <button type="button" className="button-section" value="5" onClick={() => { this.calculator(digitObj, '5'); }}>5</button>
-          <button type="button" className="button-section" value="6" onClick={() => { this.calculator(digitObj, '6'); }}>6</button>
-          <button type="button" className="button-section operator" value="-" onClick={() => { this.calculator(digitObj, '-'); }}>-</button>
-          <button type="button" className="button-section" value="1" onClick={() => { this.calculator(digitObj, '1'); }}>1</button>
-          <button type="button" className="button-section" value="2" onClick={() => { this.calculator(digitObj, '2'); }}>2</button>
-          <button type="button" className="button-section" value="3" onClick={() => { this.calculator(digitObj, '3'); }}>3</button>
-          <button type="button" className="button-section operator" value="+" onClick={() => { this.calculator(digitObj, '+'); }}>+</button>
-        </div>
-        <span className="end">
-          <button type="button" className="button-section" value="0" onClick={() => { this.calculator(digitObj, '0'); }}>0</button>
-          <button type="button" className="button-section" value="." onClick={() => { this.calculator(digitObj, '.'); }}>.</button>
-          <button type="button" className="button-section operator" value="=" onClick={() => { this.calculator(digitObj, '='); }}>=</button>
-        </span>
-      </section>
-    );
-  }
-}
+  const buttons = ['AC', '+/-', '%', '÷', '7', '8', '9', 'x', '4', '5', '6', '-', '1', '2', '3', '+'].map((btn) => (
+    (btn === 'x' || btn === '+' || btn === '-' || btn === '÷') ? (
```

Comment:
## [OPTIONAL]
- [ ] Kindly I suggest you to solve this error when the numbers are undefined
![image](https://user-images.githubusercontent.com/65068771/159122568-6684fd93-5d24-4278-a2ff-093e611e4840.png)